css-canvas-medical/styles.css

```diff
@@ -0,0 +1,244 @@
+.header {
+  display: flex;
+  margin-top: 25px;
+  margin-bottom: 10px;
+  padding-left: 25px;
+}
+
+.head-link,
+.contact-button {
+  font-size: 12px;
+  justify-self: center;
+  margin-left: 12px;
+  margin-right: 12px;
+  margin-top: 10px;
+}
+
+.contact-button {
+  background-color: rgb(29, 50, 95);
+  color: white;
+  border: 1px solid rgb(29, 50, 95);
+  border-radius: 3px;
+  padding: 7px 10px;
+  position: relative;
+  bottom: 6px;
+}
+
+.logo {
+  width: 150px;
+  margin-right: 240px;
+}
+
+p {
+  font-size: 10px;
+}
+
+p,
+button,
+.links,
+.foot-links,
+h6,
+.head-link {
+  font-family: 'Avenir', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.hero {
+  width: 100%;
+  height: 345px;
+  opacity: 0.9;
+  margin-left: 45px;
+  margin-right: 45px;
+}
+
+.get-started {
+  display: flex;
+  position: relative;
+  margin-bottom: 50px;
+  padding-right: 20px;
+  padding-left: 20px;
+}
+
+.g-s-button {
+  position: absolute;
+  left: 100px;
+  bottom: 60px;
+}
+
+.overlay {
+  background-color: rgba(0, 0, 0, 0.24);
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  margin-left: 65px;
+  margin-right: 65px;
+}
+
+h1 {
+  position: absolute;
```

Comment:
It would probably not be wise to set all `h1` elements to be absolute positioned. If this does in fact need to be absolute positioned, you should apply that styling to a class.